targets.go

```diff
@@ -121,6 +153,12 @@ func (t mapTarget) setFloat64(v float64) error {
 	return t.set(reflect.ValueOf(v))
 }
 
+var (
+	errValIndexExpectingSlice  = errors.New("expecting a slice")
+	errValIndexCanNotInitSlice = errors.New("cannot initialize a slice")
```

Comment:
Casing of this var should be `...Cannot...`?  Same for a few others in this PR.

Reply:
I'm surprised the misspell linter actually didn't tell me anything.
English is not my mother tongue, so sometimes, this kind of things can happen :). Will correct that.